src/components/Chat.js

```diff
@@ -491,13 +493,14 @@ export default class Chat extends React.Component {
           <MessageList innerRef={this._setParentScroll}>
             {parsedMessages.filter(group => group.length > 0).map(this._renderGroup)}
             {typingIndicator ?  <Message authorName={otherAuthor.name} isOwn={false} ><TypingIndicator /></Message>: null}
-            <ReactCSSTransitionGroup
-              transitionName="quickreplies-animation"
-              transitionEnterTimeout={theme.QuickReplies.animationLength}
-              transitionLeaveTimeout={theme.QuickReplies.animationLength}
-            >
-              {quickReplies && quickReplies.length > 0 ? <QuickReplies replies={quickReplies} onSelect={this._onSendQuickReply} active={true} /> : null}
-            </ReactCSSTransitionGroup>
+            {quickReplies && quickReplies.length && (
```

Comment:
Airbnb style guide [recommends](https://github.com/airbnb/javascript#comparison--shortcuts) using explicit comparison for any type but Boolean. I prefer that because JS has complex type coercion rules. But it's up to you.

Reply:
i usually go with explicit checks when a falsey check isn't enough. this check here is pretty straightforward with a general falsey check, but I have no problem being explicit as well